librz/core/analysis_tp.c

```diff
@@ -812,7 +812,7 @@ void propagate_types_among_used_variables(RzCore *core, HtUP *op_cache, RzAnalys
 #define OP_CACHE_LIMIT 8192
 
 RZ_API void rz_core_analysis_type_match(RzCore *core, RzAnalysisFunction *fcn, HtUU *loop_table) {
```

Comment:
Please add also `RZ_NONNULL` to `core` and `fcn`

Reply:
Suree